assets/js/components/ExecutionResults/ExecutionResults.test.jsx

```diff
@@ -343,4 +344,84 @@ describe('ExecutionResults', () => {
     expect(screen.getAllByText('2/2 expectations passed')).toBeTruthy();
     expect(screen.getAllByText('1/2 expectations failed')).toBeTruthy();
   });
+
+  it('given provider is VMware, should render ExecutionResults with warning banner', async () => {
+    const {
+      clusterID,
+      hostnames,
+      checks: [checkID1, checkID2],
+      loading,
+      catalog,
+      error,
+      executionLoading,
+      executionData,
+      executionError,
+      executionStarted,
+    } = prepareStateData('passing');
+
+    renderWithRouter(
```

Comment:
Allright, we don't need the `MemoryRouter` because this component doesn't have a `useParams` function in it. We can even remove the recently added `route` option from the `renderWithRouter`

Reply:
We still need to provide a `Router` context as the `BackButton` inside `ExecutionResults` uses `useNavigate()`.

https://github.com/trento-project/web/blob/main/assets/js/components/ExecutionResults/ExecutionResults.jsx#L123

https://github.com/trento-project/web/blob/main/assets/js/components/BackButton/BackButton.jsx#L7